Tools/CISupport/build-webkit-org/config.json

```diff
@@ -541,6 +542,12 @@
                     "additionalArguments": ["--no-bubblewrap-sandbox"],
                     "workernames": ["gtk-linux-bot-17"]
                   },
+                  {
+                    "name": "GTK-Linux-64bit-Release-GNOMEWebCanary-Packaging-Nightly", "factory": "BuildAndUploadBuiltProductviaSftpFactory",
+                    "platform": "gtk", "configuration": "release", "architectures": ["x86_64"],
+                    "additionalArguments": ["--no-bubblewrap-sandbox", "--prefix=/app/webkit/WebKitBuild/release/install"],
```

Comment:
```suggestion
                    "additionalArguments": ["--no-bubblewrap-sandbox", "--prefix=/app/webkit/WebKitBuild/Release/install"],
```

Reply:
Changed, I also removed `--no-bubblewrap-sandbox` so that we can catch easily any regression on the sandbox causing issues inside flatpak. I know it gets disabled automatically at runtime inside flatpak, so in order to test that it works.